config/h5n1-cattle-outbreak/auspice_config_h5n1-cattle-outbreak.json

```diff
@@ -124,7 +124,7 @@
     "map_triplicate": false,
     "color_by": "host",
     "geo_resolution": "division",
-    "distance_measure": "div"
+    "distance_measure": "__VALUE_OVERWRITTEN_BY_SNAKEMAKE_PIPELINE__"
```

Comment:
_Not asking for a change, just thinking through how this value can affect UX_

Interesting how this tightly couples the Auspice config with this version of the workflow. If the `auspice_config` rule is changed in a way that fails to edit this value, the export rule will run into a validation error, which is a great check for us. On the other hand, if someone tries to use this config in another workflow (or older version of this workflow), they will also run into a validation error.

Reply:
> the export rule will run into a validation error

This was my thinking for this decision. I could have left it as "div" but wanted to avoid silent failures. Of the two downsides to this approach you point out,

>  if someone tries to use this config in another workflow

this is fair, although hopefully the text is a big enough hint to them that they have to add something on the snakemake side! They'll get validation errors if they forget

> or older version of this workflow

given both the config and the snakemake pipeline are in version control I expect such usage to never happen, but 🤷

Reply:
> > or older version of this workflow
> 
> given both the config and the snakemake pipeline are in version control I expect such usage to never happen, but 🤷

Mostly thinking of people who copy/paste configs without being fully aware of these internal nuances 🤷‍♀️ 

But yeah, I think the validation error is clear enough in this case
```
Validating schema of 'results/h5n1-cattle-outbreak/ha/default/auspice-config.json'...
  .display_defaults.distance_measure "__VALUE_OVERWRITTEN_BY_SNAKEMAKE_PIPELINE__" failed enum validation for ["num_date", "div"]
```

Reply:
Thanks for looking through things Jover!